rs/ic_os/config/src/main.rs

```diff
@@ -367,6 +381,118 @@ pub fn main() -> Result<()> {
 
             generate_testnet_config(args, clap_args.guestos_config_json_path)
         }
+        // TODO(NODE-1519): delete UpdateGuestosConfig and UpdateHostosConfig after moved to new config format
+        Some(Commands::UpdateGuestosConfig) => update_guestos_config(),
+        Some(Commands::UpdateHostosConfig {
+            config_ini_path,
+            deployment_json_path,
+            hostos_config_json_path,
+        }) => {
```

Comment:
Should this be moved alongside `update_guestos_config`?

Reply:
Yes it should, moved!